src/main/java/com/code_intelligence/jazzer/junit/FuzzTestExecutor.java

```diff
@@ -94,7 +95,16 @@ public static FuzzTestExecutor prepare(ExtensionContext context, String maxDurat
     ArrayList<String> libFuzzerArgs = new ArrayList<>();
     libFuzzerArgs.add(argv0);
 
-    // Store the generated corpus in a per-class directory under the project root, just like cifuzz:
+    // Add passed in corpus directories (and files) at the beginning of the arguments list,
+    // as those have precedence over the JUnit specific ones added later on.
```

Comment:
"precedence" is a bit vague. It could make sense to explain that the first directory is where libFuzzer stores new inputs, whereas all the other ones just supply additional seeds and aren't written into - their particular order shouldn't really matter.

Reply:
Rephrased the comment. Is it clearer now?